vhdbuilder/packer/install-dependencies.sh

```diff
@@ -489,6 +489,7 @@ PATCHED_HYPERKUBE_IMAGES="
 1.17.7-hotfix.20200624
 1.17.7-hotfix.20200714.1
 1.17.7-hotfix.20200714.2
+1.17.7-hotfix.20200817.1
```

Comment:
thsi should fail the vhd build

Reply:
https://msazure.visualstudio.com/CloudNativeCompute/_build/results?buildId=34263402&view=logs&j=e87678bc-3b03-5f2b-4494-ebb72fdaf05d&t=67ee5b5b-46ff-51d6-88fc-b011da8c5cea